frontends/election-manager/src/utils/cast_vote_record_files.ts

```diff
@@ -350,6 +350,33 @@ export class CastVoteRecordFiles {
           throw new Error(`Line ${lineNumber}: ${errors.join('\n')}`);
         }
 
+        for (const [key, value] of Object.entries(cvr)) {
+          if (Array.isArray(value)) {
+            const votes = value as ContestOptionId[];
+            for (const vote of votes) {
+              if (vote.startsWith('write-in')) {
+                const res = await fetch('/admin/write-ins/adjudication', {
```

Comment:
If we're `POST`ing in a loop, seems like this should maybe be a bulk operation. But also… why is this data being persisted here _and_ on the server? Seems like we should have the server be the source of truth for all this?